_content/monographs/brief-intro-to-buddha-dhamma_payutto.md

```diff
@@ -0,0 +1,22 @@
+---
+title: "A Brief Introduction to Buddha-Dhamma"
+authors:
+  - payutto
+translator: "Bhikkhu Nirodho and Martin Seeger"
+external_url: "https://www.buddhistelibrary.org/en/albums/userpics/10005/a_brief_introduction_to_the_buddha_dhamma.pdf"
+drive_links:
+  - "https://drive.google.com/file/d/1VTNhR87LI30Z79TT1tMFCoGA0hiprfLD/view?usp=sharing"
+course: thai
+tags:
+  - view
+year: 2011
+publisher: "Wat Nyanavesakavan"
```

Comment:
If you're putting in monographs please add an olid with a cover image. Otherwise this can go in essays as it's pretty short